go/event/event_test.go

```diff
@@ -56,10 +58,7 @@ func TestStaticListener(t *testing.T) {
 	AddListener(func(testEvent1) { triggered = true })
 	AddListener(func(testEvent2) { t.Errorf("wrong listener type triggered") })
 	Dispatch(testEvent1{})
-
-	if !triggered {
-		t.Errorf("static listener failed to trigger")
-	}
+	assert.True(t, triggered)
```

Comment:
this will read just as `expected true, got false`, so we might want a more descriptive message here (i.e. "expected static listener to trigger")

Reply:
same applies to other `assert.True` and `assert.False` instances

Reply:
Agreed. If you are up to doing a PR against the test rewriter tool, it will be good to try and preserve the custom messages.